### ISSUE TITLE
Unit tests are failing

### ISSUE THREAD

**Issue by tmbrbr** [SAP/project-fontus#8]
The build is currently broken due to failing tests:
```
./gradlew build
```
Currently gives
```
Test result: FAILURE
Test summary: 1017 tests
        1006 succeeded
        6 failed
        5 skipped
```
It looks like most of them are SQL unit tests, the full log is available via the Github action: https://github.com/SAP/project-fontus/actions/runs/7962371074/job/21735688770

**Comment by leeN** [comment]
I think we broke them during the CCS crunch, I will have a look into them.

**Comment by leeN** [comment]
Closed in #10.
